script/Ciel/Day 13/13_01_01_CIEL13_3.txt

```diff
@@ -152,42 +169,48 @@ Can you please stop brazenly kicking up a fuss inside my head? I already feel li
 [sha:9f268dc1a4dfb18bb069a36b551b29a4bc5165e7]{
 -- Page 110, Offset 33477.
 -- それはまずい。目を閉じれば眠ってしまう。なにしろ一日まるまる眠っていない。心も体も眠りたがっている。眠ってしまえば今度こそ目覚めないと分かっている。
-No, I can't do that. If I close my eyes, then I'll end up falling asleep. After all, I haven't slept for an entire day. My body and mind are both longing for slumber. I understand that there's no waking up afterward if I do that.
+No, I can't do that.
+ It's been a full day since I last slept.
+ My body and mind are both desperate for rest.
+ If I closed my eyes, I'd be asleep in a heartbeat.
+ But I know that if I succumb to sleep now, I won't be waking up again.
 }
 [sha:7d70591b19385636bcc0f3784b298ccdab1e2ba0]{
 -- Page 110, Offset 33478.
 -- 目覚めたとしても、それは自分ではない何者かだ。
-Even if I were to wake up, I would wake up as someone that isn't myself.
+Or rather, even if I did wake up, it'd be as someone else.
```

Comment:
Or rather doesn't fit in this all too well? This is probably a job for start-of-sentence-"and"-man
>And even if...